script/install-deps

```diff
@@ -29,6 +29,26 @@
 
 . "$(dirname $0)"/common.sh
 
+set -e
+
+install_packages() {
+  if installed apt-get; then
```

Comment:
Better use 4 spaces for indention(to keep consistent with openthread projects).

Reply:
Good idea! But there are many other scripts that are using 2 spaces for indention. 
I will create another PR to fix all scripts.